packages/EasyErrorHandler/docs/laravel_intall.md

```diff
@@ -0,0 +1,126 @@
+<div align="center">
```

Comment:
Please have a look at what has been done for `EasyApiToken` or even `EasyCore` for the docs to be compatible with the online documentation, I don't think it would be a too big effort as you already have the content.

Reply:
done